scripts/ci/get_operator_github_branch.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -xe
+set +o pipefail
+
+does_operator_branch_has_image=false
+operator_branch=develop
+DOCKER_HUB_USERNAME=csiblock1
+DOCKER_HUB_PASSWORD=$csiblock_dockerhub_password
+wanted_image_tag=`echo $CI_ACTION_REF_NAME | sed 's|/|.|g'`
+export image_tags=`docker-hub tags --orgname csiblock1 --reponame ibm-block-csi-operator --all-pages | grep $wanted_image_tag | awk '{print$2}'`
+
+for tag in $image_tags
+do
+  if [[ "$tag" == "$wanted_image_tag" ]]; then
+    does_operator_branch_has_image=true
+  fi
+done
+
+if [ $does_operator_branch_has_image == "true" ]; then
+  operator_branches=`curl -H "Authorization: token $github_token" https://api.github.com/repos/IBM/ibm-block-csi-operator/branches | jq -c '.[]' | jq -r .name`
+  for branch_name in $operator_branches
+  do
+    if [ "$branch_name" == "$CI_ACTION_REF_NAME" ]; then
+      operator_branch=$CI_ACTION_REF_NAME
```

Comment:
do we really need a second var with the same value?
can't it just be a boolean of whether an identical operator branch exists or not?

Reply:
why do we need the operator branch, if we found the image?

Reply:
to check if the branch is not deleted. If the branch is deleted so it means that the content is in develop branch/latest image

Reply:
this is not so clear from current code.
could you wrap the code with functions to convey this info?
e.g.
```
operator_branch=develop
if is_operator_branch_image_exists() and is_operator_branch_exists() then
        operator_branch=triggering_branch
```